src/library/entities/Item.java

```diff
@@ -1,104 +1,112 @@
-          .append("  CallNo: ").append(CaLlNo).append("\n")
-          .append("  State:  ").append(sTaTe);
-        
-        return Sb.toString();
-    }
-
-    public Long GeTiD() {
-        return Id;
-    }
-
-    public String GeTtItLe() {
-        return TiTlE;
-    }
-
-    public ItemType GeTtYpE() {
-        return TyPe;
-    }
-
-
-    
-    public boolean Is_AvAiLaBlE() {
-        return sTaTe == iTeM_StAtE.AVAILABLE;
-    }
-
-    
-    public boolean Is_On_LoAn() {
-        return sTaTe == iTeM_StAtE.ON_LOAN;
-    }
-
-    
-    public boolean Is_DaMaGeD() {
-        return sTaTe == iTeM_StAtE.DAMAGED;
-    }
-
-    
-    public void TaKeOuT() {
-        if (sTaTe.equals(iTeM_StAtE.AVAILABLE)) 
-            sTaTe = iTeM_StAtE.ON_LOAN;
-        
-        else 
-            throw new RuntimeException(String.format("Item: cannot borrow item while item is in state: %s", sTaTe));
-        
-        
-    }
-
-
-    public void TaKeBaCk(boolean DaMaGeD) {
-        if (sTaTe.equals(iTeM_StAtE.ON_LOAN)) 
-            if (DaMaGeD) 
-                sTaTe = iTeM_StAtE.DAMAGED;            
-
-            else 
-                sTaTe = iTeM_StAtE.AVAILABLE;        
-
-        
-        else 
-            throw new RuntimeException(String.format("Item: cannot return item while item is in state: %s", sTaTe));
-                
-    }
-
-    
-    public void rEpAiR() {
-        if (sTaTe.equals(iTeM_StAtE.DAMAGED)) 
-            sTaTe = iTeM_StAtE.AVAILABLE;
-        
-        else 
-            throw new RuntimeException(String.format("Item: cannot repair while Item is in state: %s", sTaTe));
-        
-    }
-
-
-}
+package library.entities;
+
+import java.io.Serializable;
+
+@SuppressWarnings("serial")
+public class Item implements Serializable {
+
+    private ItemType type;
+
+    private String author;
+
+    private String title;
+
+    private String callNumber;
+
+    private long id;
+
+    private enum ItemState {AVAILABLE, ON_LOAN, DAMAGED, RESERVED}
+
+    private ItemState state;
+
+    public Item(String author, String title, String callNumber, ItemType itemType, long id) {
+        this.type = itemType;
+        this.author = author;
+        this.title = title;
+        this.callNumber = callNumber;
+        this.id = id;
+        this.state = ItemState.AVAILABLE;
+    }
+
+
+    public String toString() {
+        StringBuilder output = new StringBuilder();
+        output.append("Item: ").append(id).append("\n")
+            .append("  Type:   ").append(type).append("\n")
+            .append("  Title:  ").append(title).append("\n")
+            .append("  Author: ").append(author).append("\n")
+            .append("  CallNo: ").append(callNumber).append("\n")
+            .append("  State:  ").append(state);
+        return output.toString();
+    }
+
+
+    public Long getId() {
+        return id;
+    }
+
+
+    public String getTitle() {
+        return title;
+    }
+
+
+    public ItemType getType() {
```

Comment:
class diagram has this method listed as getItemType(). Please refactor/rename method name.